.gitignore

```diff
@@ -33,3 +33,7 @@ alacritty_*_source.tar.bz2
 *.msi
 *.wixobj
 *.wixpdb
+
+# Perf tools
+perf.data*
+flamegraph.svg
```

Comment:
I feel like the file is mixed aligned or something like that, could you check that?

Reply:
I hate this github viewer here. This is not an issue.